frame/staking/src/testing_utils.rs

```diff
@@ -59,6 +59,22 @@ pub fn create_funded_user<T: Config>(
 	user
 }
 
+/// Grab a funded user.
+pub fn create_funded_user_b<T: Config>(
+	string: &'static str,
+	n: u32,
+	balance_factor: crate::BalanceOf<T>,
```

Comment:
The new functions in this file that end with `_b` are fairly redundant but use `Balance` for `balance_factor` instead of u32, which plays more friendly.